client/src/App.js

```diff
@@ -13,6 +14,7 @@ class App extends Component {
         <Switch>
           <Route path="/" component={MainPage} exact />
           <Route path="/login" component={LoginPage} />
+          <Route path="/home" component={HomePage} />
```

Comment:
The home/main page will live at "/", but we can fix that once we actually assemble the page.

Reply:
Ah, missed that. Okay, I'll fix that up